tests/common/config_reload.py

```diff
@@ -108,7 +108,7 @@ def config_reload_minigraph_with_rendered_golden_config_override(
 @ignore_loganalyzer
 def config_reload(sonic_host, config_source='config_db', wait=120, start_bgp=True, start_dynamic_buffer=True,
                   safe_reload=False, wait_before_force_reload=0, wait_for_bgp=False,
-                  check_intf_up_ports=False, traffic_shift_away=False, override_config=False,
+                  check_intf_up_ports=False, traffic_shift_away=False, override_config=True,
```

Comment:
@qiluo-msft @wen587 Could you please help to check whether this change is acceptable? Since all testbed should have default golden_config_db.json with this PR, I think we can override_config by default? If not, the golden config loaded in deploy-mg stage would be override if some test case invoke config_reload to load_minigraph with default parameter

Reply:
I am good with this change. 
But this golden config may conflict with other test:
Hi @liuh-80, will your test override the golden config from Yaqiang? The golden config may need to be combined to make the test work.
https://github.com/sonic-net/sonic-mgmt/blob/master/tests/tacacs/test_authorization.py#L671
https://github.com/sonic-net/sonic-mgmt/blob/master/tests/route/test_forced_mgmt_route.py#L164

Reply:
I checked the code, the above tacacs and forced_mgmt_route test cases can work after this change.

Reply:
This method need change:

def reload_minigraph_with_golden_config(duthost, json_data, safe_reload=True):
    """
    for multi-asic/single-asic devices, we only have 1 golden_config_db.json
    """
    golden_config = "/etc/sonic/golden_config_db.json"
    duthost.copy(content=json.dumps(json_data, indent=4), dest=golden_config)
    config_reload(duthost, config_source="minigraph", safe_reload=safe_reload, override_config=True)


All code use this method will have risk.

Reply:
No need for this. https://github.com/sonic-net/sonic-mgmt/blob/master/tests/conftest.py#L2072 `core_dump_and_config_check` would save running config before test and check restore it after test